modules/ai-sno-requirements-for-installing-worker-nodes.adoc

```diff
@@ -0,0 +1,49 @@
+// This is included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+[id="ai-sno-requirements-for-installing-worker-nodes_{context}"]
+= Requirements for installing {sno} worker nodes
+
+To install a {sno} worker node, you must address the following requirements:
+
+* *Administration host:* You must have a computer to prepare the ISO and to monitor the installation.
+
+* *Production-grade server:* Installing {sno} worker nodes requires a server with sufficient resources to run {product-title} services and a production workload.
++
+.Minimum resource requirements
+[options="header"]
+|====
+
+|Profile|vCPU|Memory|Storage
+
+|Minimum|2 vCPU cores|8GB of RAM| 100GB
+
+|====
++
+[NOTE]
+====
+One vCPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio:
+
+(threads per core × cores) × sockets = vCPUs
+====
++
+The server must have a Baseboard Management Controller (BMC) when booting with virtual media.
+
+* *Networking:* The worker node server must have access to the internet or access to a local registry if it is not connected to a routable network. The worker node server must have a DHCP reservation or a static IP address and be able to access the {sno} cluster Kubernetes API, Ingress route, and cluster node domain names. You must configure the DNS to resolve the IP address to each of the following fully qualified domain names (FQDN) for the {sno} cluster:
```

Comment:
I don't think that "Ingress route" should be capitalized.

Reply:
Looks like it's copied from https://github.com/openshift/openshift-docs/blob/20ace352d73717d91949f7e05e223b445649f3e5/modules/install-sno-requirements-for-installing-on-a-single-node.adoc?plain=1#L32

Reply:
(not to say that it should not be fixed, just a remark)

Reply:
Fixed in both places